editor/src/clj/editor/gui.clj

```diff
@@ -1944,25 +1975,10 @@
                                                                   [:node-msgs :scene-node-msgs]
                                                                   [:node-overrides :template-overrides]]]
                                                    (g/connect or-scene from self to))
-                                                 (for [[from to] [[:layer-names :layer-names]
-                                                                  [:texture-gpu-textures :aux-texture-gpu-textures]
-                                                                  [:texture-infos :aux-texture-infos]
-                                                                  [:material-infos :aux-material-infos]
-                                                                  [:material-shaders :aux-material-shaders]
-                                                                  [:font-shaders :aux-font-shaders]
-                                                                  [:font-datas :aux-font-datas]
-                                                                  [:font-names :aux-font-names]
-
-                                                                  [:spine-scene-element-ids :aux-spine-scene-element-ids]
-                                                                  [:spine-scene-infos :aux-spine-scene-infos]
-                                                                  [:spine-scene-names :aux-spine-scene-names]
-
-                                                                  [:particlefx-infos :aux-particlefx-infos]
-                                                                  [:particlefx-resource-names :aux-particlefx-resource-names]
-                                                                  [:resource-names :aux-resource-names]
-                                                                  [:layout-names :aux-layout-names]
+                                                 (for [[from to] [[:template-prefix :id-prefix]
                                                                   [:current-layout :current-layout]
-                                                                  [:template-prefix :id-prefix]]]
+                                                                  [:basic-gui-scene-info :aux-basic-gui-scene-info]
+                                                                  [:costly-gui-scene-info :aux-costly-gui-scene-info]]]
```

Comment:
Connections between the `TemplateNode` and the `GuiSceneNode` it brings into the host scene.